meta-lmp-base/recipes-crypto/cryptsetup/cryptsetup_git.bb

```diff
@@ -0,0 +1,109 @@
+SUMMARY = "Manage plain dm-crypt and LUKS encrypted volumes"
```

Comment:
Carry cryptsetup with a bb append can be more adequate and easy to see the changes needed.